package.json

```diff
@@ -23,9 +23,11 @@
     "readme": "remark README.md --use ./scripts/plugin.mjs --output",
     "lint": "eslint .",
     "pretest": "npm run lint",
-    "test": "nyc mocha --async-only"
+    "test": "nyc mocha --async-only",
+    "build": "node -r ./register.js ./node_modules/.bin/webpack --mode=production"
   },
   "devDependencies": {
+    "esbuild-kit": "^0.10.0",
```

Comment:
This is unrelated to tsx:
https://www.npmjs.com/package/esbuild-kit

esbuild-kit used to be the GitHub org tsx was hosted in before it was transferred.